src/views/HomeView.vue

```diff
@@ -128,7 +158,7 @@ axios.post('/submit', bet)
     </Header>
 
     <aside class="sidebar">
-      Sidebar
+        <SidebarItem v-for="link in linksWithIcon" :text="link.text" :icon="link.icon">{{ link }}</SidebarItem>
     </aside>
```

Comment:
Remember that every `v-for` need a `key`. And also that any DOM tag that has more than 2 attrs needs to be broken across lines. And the last thing is that your attributes should be (generally) Vue tags then attributes (alpha) then events.